modules/react/icon/lib/Graphic.tsx

```diff
@@ -26,7 +26,17 @@ export interface GraphicStyles {
   grow?: boolean;
 }
 
-export interface GraphicProps extends GraphicStyles, Pick<SvgProps, 'shouldMirror' | 'cs'> {
+export interface CanvasGraphic {
+  name: string;
+  type: CanvasIconTypes;
+  svg?: string;
+  filename: string;
+  category?: string;
+  tags?: string[];
+  url?: string;
+}
```

Comment:
Rewrote the type interface to make URL and svg optional. cc @RayRedGoose

Reply:
I'm still seeing type issues with it. 🤔

Reply:
`Graphic` only expects a `src` prop which is an object. I've updated the type to have an additional property of `url`.  My thought was that the graphic package would also export a object not just a URL. If that's not the case, I can add an additional `url` prop to Graphic to support the URL.